packages/sdk/package.json

```diff
@@ -9,8 +9,9 @@
   ],
   "scripts": {
     "all": "yarn clean && yarn build && yarn test && yarn lint:fix && yarn lint",
+    "prebuild": "npx ts-node scripts/generateTokenList.ts",
     "build": "tsc -p tsconfig.json",
-    "clean": "rimraf dist/ ./tsconfig.tsbuildinfo",
+    "clean": "rimraf dist/ ./tsconfig.tsbuildinfo && rimraf generated/",
```

Comment:
I don't think you need to call `rimraf` twice here

Reply:
Good catch